src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -259,6 +270,23 @@ private HearingEntity saveHearingDetails(HearingRequest createHearingRequest) {
         return hearingRepository.save(hearingEntity);
     }
 
+    private void savePartyRelationshipDetails(HearingRequest createHearingRequest, HearingEntity hearingEntity) {
+
+        final List<PartyDetails> partyDetails = createHearingRequest.getPartyDetails();
+        final List<HearingPartyEntity> hearingParties = hearingEntity.getLatestCaseHearingRequest().getHearingParties();
```

Comment:
Could `hearingParties` be null here ? If so what would we do - just not save any partyRelationshipDetails to DB ?

Reply:
There can be no parties according to the spec, and relationships are submitted against parties in the request. We should do nothing in this scenario. On line 302 we could harden it by:
```
if (!Collections.isEmpty(partyDetails) && !Collections.isEmpty(hearingParties))
```
(Note the `isEmpty()` util method also checks for null))

Reply:
Fixed in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/768e22b379256ea506790b5fc818777349748c05